src/api.js

```diff
@@ -48,7 +48,8 @@ function getPushDetails(eventData) {
   // parsing the ref refs/heads/<branch-name> is the most reliable way
   // to get a branch name
   let branch = ref.split('/').slice(2).join('/');
-  return {
+  let isTagEvent = ref.split('/')[1] === 'tags';
+  let details = {
     'event.base.ref': ref,
     'event.base.repo.branch': branch,
```

Comment:
I think we should leave this out for tags -- there really is no branch in this case!